src/client.rs

```diff
@@ -1,19 +1,23 @@
-
 mod kinto_http;
 mod signatures;
 
-use log::{debug};
-use kinto_http::{get_changeset, KintoObject, KintoError};
-use signatures::{DefaultVerifier};
+use kinto_http::{get_changeset, KintoError, KintoObject};
+use log::debug;
 pub use signatures::{SignatureError, Verification};
 
+#[cfg(feature = "openssl_verifier")]
+use crate::client::signatures::openssl_verifier::OpenSSLVerifier;
+
+#[cfg(feature = "dummy_verifier")]
+use crate::client::signatures::dummy_verifier::DummyVerifier;
```

Comment:
What happens if you write as `openssl_verifier::OpenSSLVerifier as DefaultVerifier`  in both places?
Couldn't we have `Box::new(DefaultVerifier {})` ?

I'd like to keep this `cfg(feature)` [as DRY as possible](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Reply:
We cannot bind two different imports to the same name

Reply:
yes done